packages/backend-for-frontend/src/model/errors.ts

```diff
@@ -166,6 +167,16 @@ export function tenantNotFound(tenantId: string): ApiError<ErrorCodes> {
   });
 }
 
+export function tenantSelfcareNotFound(
+  selfcareId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Tenant with Selfcare ID ${selfcareId} forbidden`,
+    code: "tenantSelfcareNotFound",
+    title: "Tenant forbidden",
```

Comment:
minor, semantically the error is "tenant not found", it's the specific endpoint that interprets the error as "forbidden"

```suggestion
    detail: `Tenant with Selfcare ID ${selfcareId} not found`,
    code: "tenantSelfcareNotFound",
    title: "Tenant not found",
```